app/controllers/friends_controller.rb

```diff
@@ -1,6 +1,12 @@
 class FriendsController < ApplicationController
   def index
-    @friends = UserFacade.friends_of(current_user)
     @user = UserFacade.search(params[:email]) if params[:email]
+    @accepted_friends = UserFacade.relationships_filter(current_user, "accepted")
+    @rejected_friends = UserFacade.relationships_filter(current_user, "rejected")
```

Comment:
Why are we doing a lookup for rejected and pending friends as well on this page?